pinot-query-runtime/src/main/java/org/apache/pinot/query/runtime/operator/BaseMailboxReceiveOperator.java

```diff
@@ -109,19 +97,18 @@ public List<MultiStageOperator> getChildOperators() {
   @Override
   public void close() {
     super.close();
-    releaseRemainingMailboxes();
+    cancelRemainingMailboxes();
   }
 
   @Override
   public void cancel(Throwable t) {
     super.cancel(t);
-    releaseRemainingMailboxes();
+    cancelRemainingMailboxes();
```

Comment:
```suggestion
    cancelRemainingMailboxes(t);
```

Reply:
Currently it is used by both `close()` and `cancel(Throwable t)`. Currently seems I can remove the one under `close()`, but if we support any kind of early termination in the future, `cancel` won't always be called.